README.md

```diff
@@ -267,6 +269,25 @@ OPTIONS:
         --ignore <pattern>    Ignore the crates matched by glob
 ```
 
+### Plan
+
+Print the plan for publishing. The output will contain the names and versions of the workspace crates according
+to the order in which they should be published.
+
+```
+USAGE:
+    cargo workspaces plan [OPTIONS]
+
+OPTIONS:
+    -h, --help
+            Print help information
+
+PUBLISH PLAN OPTIONS:
+    --check-published        Check if the crates are already published and include the information in the output
```

Comment:
@popzxc I am working on getting this merged. I think this option would be nice as `--skip-published` (which skips the printing of already published crates). Do you have any issues if I change it like that?

Reply:
Sure